bin/common.ml

```diff
@@ -1505,8 +1515,11 @@ let envs =
         ~doc:"If different than $(b,0), ANSI colors should be enabled no matter what."
         "CLICOLOR_FORCE"
     ; info
-        "DUNE_CACHE_ROOT"
         ~doc:"If set, determines the location of the machine-global shared cache."
+        "DUNE_CACHE_ROOT"
+    ; info
+        ~doc:"If set, determines the location of all the different caches used by dune."
+        "DUNE_CACHE_HOME"
```

Comment:
I think a clarification of the docs is in order, explaining briefly 

- for `DUNE_CACHE_ROOT` what kind of thing goes on the `db` and so what exactly this is configuring
- that `DUNE_CACHE_ROOT` takes precedence over `DUNE_CACHE_HOME`
- that these env vars take precedence over settints in the config